tools/pylint-extensions/pylint_custom_plugin/ci.yml

```diff
@@ -0,0 +1,47 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/pylint-extensions/pylint_custom_plugin
+
+pr:
+  branches:
+    include:
+      - main
+      - feature/*
+      - release/*
+      - hotfix/*
+  paths:
+    include:
+      - tools/pylint-extensions/pylint_custom_plugin
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tool-python.yml
+  parameters:
+    PythonVersion: '3.10.2'
+    PackagePath: 'tools/pylint-extensions/pylint_custom_plugin'
+    FeedName: 'public/azure-sdk-for-python'
+    ArtifactName: 'azsdkpylintplugin'
+    PackageName: 'Azure SDK Pylint Plugin'
+    TestSteps:
+    - template: /eng/pipelines/templates/steps/use-python-version.yml
+      parameters:
+        versionSpec: '3.10.2'
```

Comment:
PythonVersion

Reply:
What? [no it definitely takes versionSpec](https://github.com/Azure/azure-sdk-tools/blob/2218892688c1c2cd46e118b16b951233b667d76a/eng/pipelines/templates/steps/use-python-version.yml#L2)

Or are you saying we should make this a variable, I'll do that 👍

Reply:
I'm just calling out we already have a variable with the same value above that we should use here.

Reply:
The code comment no longer highlights exactly which line you're talking about, so it's super confusing :P

Reply:
You got it.